### DISCUSSION TITLE
[Feature request] Markdown/code editor with consistent/same font size for headers

### DISCUSSION THREAD

**Discussion by theontho** [standardnotes/forum#2493] [Suggestions]
**Is your feature request related to a problem? Please describe.**
I would like to be able to tell the markdown / code editors to not make things like headers have a larger font size. I don't see any settings page for specific extensions to do so although. 

It would also be nice to be able to provide my own syntax highlighting rule sets for the code editor for example, that would be compatible with some other widespread format, like Sublime Text's, TextMate or VSCode's.  

I understand that not everyone would like my changes, which is why I would like them as options.

**Comment by JaspalSuri** [comment]
Hi @theontho,

Thank you for the feedback! 

I've put in a request to provide the option to have the headings of the Markdown Editors appear in the same size as the the other non-heading text. 

[The Code Editor](https://standardnotes.org/extensions/code-editor) is derived from [CodeMirror](https://github.com/codemirror/codemirror), so you can request the ability to add your own syntax highlighting by creating a [thread on latter's repo](https://github.com/codemirror/CodeMirror/issues). 🙂

**Comment by theodorechu** [comment]
You could also try the **unofficial** [Append Editor](https://demo.appendeditor.com) ([source](https://github.com/theodorechu/append-editor)). You may be interested in the Monaco editing mode. Go to the Settings on top > Monaco editing mode > Save.

You can use the custom CSS feature to import a custom highlight.js stylesheet.
